server/controller/handler/handlers.go

```diff
@@ -66,33 +56,27 @@ func FileHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	key := keys[0]
 	var path = string(key)
-	log.Printf(path)
-	pathParser(w, path)
+	file, err := pathParser(path)
+	if err != nil {
+		log.Error(err)
```

Comment:
with logging the error you have to return some formatted error in response also

Reply:
Can you please exemplify @chandankumar4 ?

Reply:
just you have to return the error message in response also

Reply:
(write the error message in response)
log.Error(err)
return

Reply:
Done!